docs/admin_api/rooms.md

```diff
@@ -511,3 +512,55 @@ optionally be specified, e.g.:
         "user_id": "@foo:example.com"
     }
 ```
+
+# Forward Extremities Admin API
+
+Enables querying and deleting forward extremities from rooms. When a lot of forward
+extremities accumulate in a room, performance can become degraded.
+
+When using this API endpoint to delete any extra forward extremities for a room, 
+the server does not need to be restarted as the relevant caches will be cleared 
+in the API call.
+
+## Check for forward extremities
+
+To check the status of forward extremities for a room:
```

Comment:
What sort of situations would you use this in? Is the state group useful?

Reply:
No, probably not particularly, except maybe for developers. I did think of returning just a count, but it felt that since the information "is there", it felt unnecessary to not return results as well.

My understanding of the needs for this API endpoint from an operational point of view is mainly to see the count, though I would maybe check with the EMS ops team first to confirm.

Reply:
I suspect that's just internal state then and not worth returning. @erikjohnston do you see any value in returning those?

Reply:
I've sometimes used the state group as a proxy for how big the state difference was between extremities were, but it's a bad heuristic. I think a more useful one would probably be `depth` and `received_ts`, as that'll give you a better way of looking for stuck extremities

Reply:
Do we consider those to be useful in this admin API? I guess the options are:

1) keep as is
2) remove `state_group`
3) remove `state_group`, add `depth` and `received_ts`
4) remove everything, just return a count
5) something else? :)

I don't have enough experience on the extremities to vote really.

Reply:
I think adding `depth` and `received_ts`  is probably the right thing